docs/detections/rules-ui-manage.asciidoc

```diff
@@ -158,6 +161,20 @@ NOTE: When duplicating a rule that has exceptions, you can choose to duplicate t
 * Select the *All actions* menu (*...*) on a rule, then select an action.
 * Select all the rules you want to modify, then select an action from the *Bulk actions* menu.
 * To enable or disable a single rule, switch on the rule's *Enabled* toggle.
+* To <<snooze-rule-actions,snooze>> actions for rules, click the bell icon.
+
+[float]
+[[snooze-rule-actions]]
+=== Snooze rule actions
+
+Instead of turning off rules to stop alert notifications, snooze rule actions for a defined period of time. When you snooze rule actions, the rule continues to run on its defined schedule but alerts won't trigger any actions and notifications won't be sent. 
```

Comment:
```suggestion
Instead of turning off rules to stop alert notifications, snooze rule actions for a defined period of time. When you snooze rule actions, the rule continues to run on its defined schedule but alerts won't trigger any actions and notifications aren't sent. 
```